### PR TITLE
Improve design of interestgroup page

### PR DESCRIPTION
**Pull request by itsisak**
# Description

Quick fix to make the page look prettier. Took inspiration from the joblistings page. The hover effect especially is the same. 

# Result

If you've made visual changes, please include **before** and **after** images, preferably with a description. Make sure they do not contain any real user information.

**before**

https://github.com/webkom/lego-webapp/assets/45620425/69173208-2484-40b9-82ff-7ee8ab3c3a74

**after**
(Note: not all logos is actually the same, this is just for demonstration purposes since very few would load)

https://github.com/webkom/lego-webapp/assets/45620425/6ec44948-7686-4ccb-b522-73257587bc32

# Testing

- [X] I have thoroughly tested my changes.

---

Resolves ABA-246

### CODE REVIEW

File: app/routes/interestgroups/components/InterestGroupList.tsx [hunk 1]
```diff
@@ -38,32 +38,44 @@ const InterestGroupList = () => {
...
           </Link>
         )}
-      </div>
-      <div className="groups">
-        {activeGroups.map((g) => (
-          <InterestGroupComponent group={g} key={g.id} active={true} />
-        ))}
+      </Flex>
+
+      {activeGroups.map((g) => (
```
Comment (norbye) [inline app/routes/interestgroups/components/InterestGroupList.tsx:60]:
```suggestion
      {activeGroups.map((group) => (
```

File: app/routes/interestgroups/components/InterestGroupList.tsx [hunk 2]
```diff
@@ -38,32 +38,44 @@ const InterestGroupList = () => {
...
+      ))}
+
+      <div className={styles.inactiveHeader}>
         <h2>Ikke-aktive interessegrupper</h2>
         <p>
           Send gjerne e-post til
-          <a href="mailTo:interessegrupper@abakus.no"> oss </a> hvis du ønsker å
-          åpne en av disse igjen!
+          <a href="mailTo:interessegrupper@abakus.no">
+            {' '}
```
Comment (norbye) [inline app/routes/interestgroups/components/InterestGroupList.tsx:69]: Dunno if it makes any difference, but it feels more legal to add this space outside of the a-tag than inside 🤷

File: app/routes/interestgroups/components/InterestGroupList.tsx [hunk 3]
```diff
@@ -38,32 +38,44 @@ const InterestGroupList = () => {
...
+            interessegrupper@abakus.no{' '}
```
Comment (norbye) [inline app/routes/interestgroups/components/InterestGroupList.tsx:70]: Here you have two empty spaces after each other so your browser will only render one - might as well delete the other

File: app/routes/interestgroups/components/InterestGroupList.tsx [hunk 4]
```diff
@@ -38,32 +38,44 @@ const InterestGroupList = () => {
...
+          </a>{' '}
+          hvis du ønsker å åpne en av disse igjen!
         </p>
-        {notActiveGroups.map((g) => (
-          <InterestGroupComponent group={g} key={g.id} active={false} />
-        ))}
       </div>
+
+      {notActiveGroups.map((g) => (
```
Comment (norbye) [inline app/routes/interestgroups/components/InterestGroupList.tsx:76]:
```suggestion
      {notActiveGroups.map((group) => (
```

File: app/routes/interestgroups/components/InterestGroup.tsx [hunk 5]
```diff
@@ -1,52 +1,40 @@
 import { Flex } from '@webkom/lego-bricks';
+import cx from 'classnames';
 import { Link } from 'react-router-dom';
 import { Image } from 'app/components/Image';
 import styles from './InterestGroup.css';
 import type { Group } from 'app/models';
 // TODO: rather handle this in the backend
 const SAMPLE_LOGO = 'https://i.imgur.com/Is9VKjb.jpg';
```
Comment (norbye) [inline app/routes/interestgroups/components/InterestGroup.tsx:8]: Dafaq is this thing? It's not your doing so leave it if you will - I just couldn't not comment on it.. Wouldn't it be a lot better to use the abakus-logo with just the red ball for interest groups that are missing logos than some random imgur image?

**Review by norbye — APPROVED** [review]
Awesome! This has been looking crap for way too long(: Love the visual stuff and had some quick knitpicks on the code - although mostly stuff that was there before you started editing

Have you tested to see how the changes look in light mode?

**Review by haukkagu — APPROVED** [review]
Looks A LOT cleaner. Good job :fire:

File: app/routes/interestgroups/components/InterestGroup.tsx (hunk 5 above)
Comment (itsisak) [reply app/routes/interestgroups/components/InterestGroup.tsx:8]: Yes, I agree, it does not show even when the normal logos does not load in, so I did not think of it, but I will change it!

**Comment by itsisak** [comment]
@norbye Lightmode and mobileview. Now also with abakus icon as alternate image. 

**lightmode**

https://github.com/webkom/lego-webapp/assets/45620425/74356407-7d74-4a63-8539-10a1a3304b4e

**mobileview**

https://github.com/webkom/lego-webapp/assets/45620425/2544513f-a23d-45ec-a932-a3ad4725efb9

**Review by ivarnakken — APPROVED** [review]
Looks much better! Thansk! 🏅

File: app/routes/interestgroups/components/InterestGroup.css [hunk 6]
```diff
@@ -17,16 +18,46 @@
...
+    background-color: rgba(255, 0, 0, var(--color-red-hover-alpha));
+  }
+}
+
+.inactiveListItem:hover {
+  background-color: rgba(123, 123, 123, var(--color-red-hover-alpha));
+}
+
+.listItemName {
+  font-weight: 600;
```
[the remark this answers is not in the document]
Comment (itsisak) [reply app/routes/interestgroups/components/InterestGroup.css:45]: This was on the joblisting headers, but I see now that it actually does nothing, so I will remove it.

File: app/routes/interestgroups/components/InterestGroup.css [hunk 7]
```diff
@@ -17,16 +18,46 @@
...
 .listItem {
-  margin: 8px;
-  padding-left: 4px;
-  background: var(--color-white);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 10px;
+  color: var(--lego-font-color);
+  border-radius: var(--border-radius-lg);
```
[the remark this answers is not in the document]
Comment (itsisak) [reply app/routes/interestgroups/components/InterestGroup.css:30]: Yes, it is the same as joblistings.

**Pull request merged**